Description: Disable deps wasm-bindgen{,-test}
 The latter isn't uploaded yet, the former isn't used directly but presumably
 to provide support for the latter.
Last-Update: 2024-08-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -89,2 +89,2 @@
-[target."cfg(target_arch = \"wasm32\")".dependencies.wasm-bindgen]
-version = "0.2"
+#[target."cfg(target_arch = \"wasm32\")".dependencies.wasm-bindgen]
+#version = "0.2"
@@ -92,2 +92,2 @@
-[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-test]
-version = "0.3"
+#[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-test]
+#version = "0.3"
